src/app.ts

```diff
@@ -24,6 +24,7 @@ async function app() {
       path.join('.', 'build', 'input', 'resources')
     )
     .option('-d, --debug', 'output extra debugging information')
+    .version(getVersion(), '-v, --version', 'print SUSHI version')
```

Comment:
This is what made it take longer than expected.  I did not realize that you have to use a special `version` function -- I was trying to use the normal `option` function to do it.